backend/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "engines": {
-    "node": "12.15.0",
```

Comment:
why did we enforce a specific node version? Removing it could cause some problems I think
On my code I changed it to `12.x`

Reply:
I enforced just because I thought it's a good practice to be specific to ensure reproducibility.

Reply:
I think it is a good idea but we should stay to a major version if we wants  to specify a version.
There are too many versions of node, and most of the time nobody has the same.
If we want to enforce a specific version it will be painful because it will force us to use a docker image anytime we want to work on the backend. 
This is a choice we can make but I think it is too complicated for now.
We could wait until we have a problem with a version that would force us to define a specific version?

Maybe we can define a version that force use to be with the 12 like I did, or maybe we can specify it to allow 12 and above?

What do you think?

Reply:
I'm by no means an expert in Node so please choose whatever you think is most reasonable.

Reply:
I think we can stick with `12.*` for now since it is the current main node.js version. 
So this is mostly the version any node developer will use (if they have not an older version due to another project ^^)

Reply:
Sticking to `12.x` seems like a good compromise. Will update my PR